packages/dropdown/src/react/index.js

```diff
@@ -319,3 +354,16 @@ export const appearances = vars.appearances
 export const sizes = vars.sizes
 
 export default Dropdown
+
+function canUseDOM() {
+  return !!(
+    typeof window !== 'undefined' &&
+    typeof window.document !== 'undefined' &&
+    typeof window.document.createElement !== 'undefined'
+  )
+}
+
+function createUniversalPortal() {
+  if (!canUseDOM()) return null
+  return ReactDOM.createPortal(...arguments)
+}
```

Comment:
These two function were copied over from Position. Wondering if we should add to utils package?

Reply:
i'd propose using [exenv](https://www.npmjs.com/package/exenv) for the first function. and i do think the portal code could be moved to the utils package